### PR TITLE
Feature/emails nav

### PR DESCRIPTION
**Pull request by AlexNGerman**

### CODE REVIEW

File: app/views/devise/mailer/reset_password_instructions.html.erb [hunk 1]
```diff
@@ -1,8 +1,94 @@
...
+        <span style="font-family: Arial, Tahoma, Geneva, 'Times New Roman', sans-serif; font-size: 14px; color: #272727;">
+        If you didn't register for SISU, please <%= link_to 'contact SISU here', '#', target: '_blank', style: "color: #5B8EFF;" %>.</span>
+        <div style="height: 5px; line-height: 5px; font-size: 7px;">&nbsp;</div>
+      </div>
+    </td>
+  </tr>
+</table>
+
+
+<!--[if (gte mso 9)|(IE)]>
```
Comment (anerhan) [inline app/views/devise/mailer/reset_password_instructions.html.erb]: Are you shure this tags need for ie9?

File: app/views/devise/mailer/reset_password_instructions.html.erb [hunk 2]
```diff
@@ -1,8 +1,94 @@
...
+    }
+
+    table {
+      border-collapse: collapse;
+    }
+
+  </style>
+</head>
+<body style="font-family: 'Times New Roman', Arial, sans-serif; color: #000000; background-color: #EDF0F5;">
+<!--[if (gte mso 9)|(IE)]>
```
Comment (anerhan) [inline app/views/devise/mailer/reset_password_instructions.html.erb]: And This need?

**Comment by anerhan** [comment]
👍

File: app/views/devise/mailer/reset_password_instructions.html.erb (hunk 1 above)
Comment (AlexNGerman) [reply app/views/devise/mailer/reset_password_instructions.html.erb]: This is not only for browsers. It's also need for emails clients like Outlook 2010+ and others. 

Email layouts != browsers layouts %)

File: app/views/devise/mailer/reset_password_instructions.html.erb (hunk 2 above)
Comment (AlexNGerman) [reply app/views/devise/mailer/reset_password_instructions.html.erb]: read above.

 Also it's draft commit. I tested squash of commits.

**Pull request merged**